tests/downloader/src/main.cpp

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/tc_util.h>
+#include <zephyr/ztest.h>
+
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include <aos/common/tools/log.hpp>
+#include <aos/common/tools/timer.hpp>
+
+#include "downloader/downloader.hpp"
+
+using namespace aos;
+
+#define UNUSED(x) (void)(x)
```

Comment:
There is macro ARG_UNUSED in zephyr.

Reply:
Remove, because this macros does not uses